docs/pages/examples/inversion.rst

```diff
@@ -0,0 +1,31 @@
+.. _examples-inversion:
+
+Inversion of signatures
+#######################
+
+We show below a simple example of signature inversion. A crucial parameter is the :attr:`depth` chosen for the signature: the function :func:`signatory.invert_signature` reconstructs a piecewise linear path with :attr:`depth` + 1 points that approximates the original path the signature was computed on.
+
+.. code-block:: python
+
+    import numpy as np
+    import torch
+    import signatory
+
+    # Create a path consisting in a half circle
+    time = np.linspace(0, 1, 10)
+    path = torch.from_numpy(np.stack([np.cos(np.pi * time), np.sin(np.pi * time)])).T.unsqueeze(0)
```

Comment:
Doing this in just PyTorch would be preferred: `torch.linspace` etc.

Reply:
Done !